### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by olliethomas**

### CODE REVIEW

**Review by thomkeh — APPROVED** [review]
Do people capitalize "Issue"?

**Comment by olliethomas** [comment]
I don't know :(

**Comment by olliethomas** [comment]
I capitalized because it's a thing and I feel fine writing Pull Request or PR instead of pull request and pr, so I was extending that to Issue vs issue. This may be wrong

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+1. Fork the repository, clone and `cd` to the newly cloned directory.
+2. Setup and activate a new environment
+3. Install the developer tools `pip install -e .[dev]`
+4. (Optional) Install the pre-commit tools `pre-commit install`
+5. Write your super cool new thing
+6. (Optional) Check if your code would pass CI by running `python check_all.py` locally.
+
+## CI
+We use GitHub actions to test our code and make sure that we're keeping up the standards we set for ourselves.
+Before your code can be accepted it must be formatted with Black, conform to PyLint, covered by tests, include MyPy type hints and be docuemnted.
```
Comment (olliethomas) [inline CONTRIBUTING.md:18]:
```suggestion
Before your code can be accepted it must be formatted with Black, conform to PyLint, covered by tests, include mypy type hints and be documented.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+## Getting setup
+1. Fork the repository, clone and `cd` to the newly cloned directory.
+2. Setup and activate a new environment
+3. Install the developer tools `pip install -e .[dev]`
+4. (Optional) Install the pre-commit tools `pre-commit install`
+5. Write your super cool new thing
+6. (Optional) Check if your code would pass CI by running `python check_all.py` locally.
+
+## CI
+We use GitHub actions to test our code and make sure that we're keeping up the standards we set for ourselves.
```
Comment (thomkeh) [inline CONTRIBUTING.md:17]:
```suggestion
We use GitHub Actions to test our code and make sure that we're keeping up the standards we set for ourselves.
```

**Pull request merged**